samples/subsys/ipc/rpmsg_demo/src/rsc_table.cpp

```diff
@@ -0,0 +1,63 @@
+#include "rsc_table.hpp"
+#include <openamp/virtio.h>
+#include <stddef.h>
+#include <device.h>
+
+#define RSC_VDEV_FEATURE_NS (1) /* Support name service announcement */
+#define RSC_VDEV_NUM_VRINGS (2)
+
+#define VDEV0_VRING_BASE (DT_REG_ADDR(DT_NODELABEL(vdev0vring0)))
+#define VDEV1_VRING_BASE (DT_REG_ADDR(DT_NODELABEL(vdev0vring1)))
+
+#define VRING_ALIGN (0x1000U)
```

Comment:
This appears to be different from the existing `resource_table` sample. Maybe make this a `Kconfig` variable?